fml/thread_local.h

```diff
@@ -60,22 +61,18 @@ class ThreadLocal {
 
   static inline void ThreadLocalDestroy(void* value) {
     FML_CHECK(value != nullptr);
-    auto box = reinterpret_cast<Box*>(value);
+    auto* box = reinterpret_cast<Box*>(value);
     box->DestroyValue();
     delete box;
   }
 
  public:
-  ThreadLocal() : ThreadLocal(nullptr) {}
```

Comment:
This change didn't seem to take into account #else branch further down at line 112